rspace/src/test/scala/coop/rchain/rspace/concurrent/MultiLockTest.scala

```diff
@@ -93,11 +94,13 @@ class MultiLockTest extends FlatSpec with Matchers {
 
     def acquire(seq: List[String]) =
       tested.acquire(seq) {
-        for {
-          k <- seq
-          v = m.getOrElse(k, 0) + 1
-          _ = m.put(k, v)
-        } yield ()
+        seq.pure[IO].map { keys =>
```

Comment:
Why are you using `cats.IO` in tests if you use `monix Task` in production?

Reply:
wrong person to ask...

Reply:
I think you should be using the same effect. `Task` and `IO` should have similar semantics but it may not be true in every case which means you can have tests pass but have different behavior in production.

Reply:
agreed. but this PR does not address this issue :)

Reply:
Sure, just sth I noticed and drew my attention.